package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinecone-database/pinecone",
-  "version": "0.1.5",
```

Comment:
Can you revert this change? CI handles version bumps now.

Reply:
Sweet!!